cookbook/agent_concepts/knowledge/vector_db/pgvector/pgvector_db.py

```diff
@@ -12,9 +12,9 @@
 knowledge.add_content(
     name="Recipes",
     url="https://agno-public.s3.amazonaws.com/recipes/ThaiRecipes.pdf",
-    metadata={"user_tag": "Recipes from website"},
+    metadata={"doc_type": "recipe_book"},
 )
 
 knowledge.remove_vectors_by_name("Recipes")
 # or
-knowledge.remove_vectors_by_metadata({"user_tag": "Recipes from website"})
+knowledge.remove_vectors_by_metadata({"doc_type": "recipe_book"})
```

Comment:
should we keep this knowledge pattern of removing?
or the vector db one-

```py
vector_db.delete_by_metadata({"doc_type": "recipe_book"})
```
in any case lets stick to one? probably the vector db one for all cookbooks